src/social-network/defaults/injectComments.tsx

```diff
@@ -16,28 +16,28 @@ export function injectPostCommentsDefault(config: injectPostCommentsDefaultConfi
     return function injectPostComments(this: SocialNetworkUI, current: PostInfo) {
         const selector = current.commentsSelector
         const commentWatcher = new MutationObserverWatcher(selector, current.rootNode)
-            .useForeach(commentNode => {
-                const commentRef = new ValueRef(commentNode.current.innerText)
+            .useForeach((commentNode, key, meta) => {
+                const commentRef = new ValueRef(commentNode.innerText)
```

Comment:
property 'innerText' does not exist on 'commentNode'. did you forgot to reference in 'current'?